ffi_tests_output.txt

```diff
@@ -311,29 +301,14 @@ Running tests/ffi/regress_jump_to_frame_test.dart
 
 
 ===== CRASH =====
-si_signo=Bus error: 10(10), si_code=1, si_addr=0x600002058e70
-version=3.0.6 (stable) (Fri Jul 21 14:08:38 2023 -0700) on "macos_simarm64"
-pid=51357, thread=11267, isolate_group=main(0x13d01aa00), isolate=main(0x13d02a800)
+si_signo=Segmentation fault: 11(11), si_code=2, si_addr=0x10
```

Comment:
Interesting that we're now dereffing null (well offset 0x10 from zero).

Reply:
This is another corner case: there is no transition `next` above the handler frame so we can't unwind down to `next->previous()` and instead will have to effectively unwind down to the top transition.